recipes/libsvm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.21" %}
+{% set flatversion = version.replace(".", "") %}
+
+package:
+  name: libsvm
+  version: {{ version }}
+
+source:
+  fn: libsvm-{{ flatversion }}.tar.gz
+  url: https://github.com/cjlin1/libsvm/archive/v{{ flatversion }}.tar.gz
+  md5: acff2c058b7cb38a135fa5d0b4f39e43
+
+  patches:
+    - win32.patch  # [win32]
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}                    # [win]
```

Comment:
Because it isn't actually needed for building? This pattern of setting Python as a build requirement when it isn't one seems strange to me. The correct MSVC versions are chosen assuming CONDA_PY is correctly set, *without* it being included. It seems more like a bug in the resolution of which Python is being used, or at least a quirk. I feel like, since there are so many recipes we need to pin to particular compilers, this should be simpler and more 'obvious'.

Also, adding it to build (and test) still doesn't work 😖